internal/buildscripts/packaging/installer/install.sh

```diff
@@ -677,7 +677,7 @@ Options:
                                         (default: https://ingest.REALM.signalfx.com/v2/trace)
   --uninstall                           Removes the Splunk OpenTelemetry Collector for Linux.
   --with[out]-fluentd                   Whether to install and configure fluentd to forward log events to the collector.
-                                        (default: --with-fluentd)
+                                        (default: --without-fluentd)
```

Comment:
So --without-fluentd is preserved for backward compatibility? What happens if both are used? Is evaluation order based? Just thinking about conflicts.

Reply:
- Correct, similar to how we currently support `--with-fluentd` even though it was the default option.
- The last option specified in the installation command will be used since we parse the options serially.